autopush/db.py

```diff
@@ -606,6 +650,64 @@ def drop_user(self, uaid):
         return self.table.delete_item(uaid=huaid,
                                       expected={"uaid__eq": huaid})
 
+    def delete_uaids(self, uaids):
+        """Issue a batch delete call for the given uaids
+
+        :type uaids: List[str]
+
+        """
+        with self.table.batch_write() as batch:
+            for uaid in uaids:
+                batch.delete_item(uaid=uaid)
+
+    def drop_old_users(self, months_ago=2):
+        """Drops user records that have no recent connection
+
+        Utilizes the last_connect index to locate users that haven't
+        connected in the given time-frame.
+
+        The caller must iterate through this generator to trigger batch
+        delete calls of the ``batch_size`` provided. Caller should wait as
```

Comment:
Probably should remove these, since we don't actually take `batch_size` as a parameter. That or specify that batches are 25 items.